src/server/services/registrar.js

```diff
@@ -197,8 +197,8 @@ async function getShortname(offeredTermCode, classSectionID) {
       ''
     )}-${secNum.replace(/^0+/g, '')}`;
 
-    registrarDebug(`getShortname: returning ${shortname}`);
-    return shortname;
+    registrarDebug(`getShortname: returning { ${shortname}, ${subArea}`);
+    return { shortname, subjectArea: subArea };
```

Comment:
@chern Since this is now returning an object, make sure to update the JSdoc for the return type: https://github.com/ucla/lti-media-resources/blob/3fa6b24c7c0b077e13d902542c6cbc6837a20d56/src/server/services/registrar.js#L137